Gemfile

```diff
@@ -13,6 +13,8 @@ gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 # Use Active Model has_secure_password
 gem 'bcrypt', '~> 3.1.7'
+# Brings convention over configuration to your JSON generation
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
No, never use this gem!!! Please use `jsonapi-serializer`, fork of `fast_json` instead.

Reply:
Replaced with "jsonapi-serialize"